### DISCUSSION TITLE
深度之家建议增加回复功能

### DISCUSSION THREAD

**Discussion by Lichenkass** [linuxdeepin/developer-center#5137] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

之前在深度之家提交几个bug及需求，官方人员在回复后，即使想回复也回复不了，还需要再次提交bug，类似下面这种,
![image](https://github.com/linuxdeepin/developer-center/assets/88034509/8ce05edc-8638-48fd-81cb-cc8f035c9dfa)
出现POT无法使用的原因是因为缺少libicu67依赖导致tesseract-ocr无法安装，POT 1.13.1版本依赖包括tesseract-ocr,商店的1.10.1版本是因为打包发现无法正常在V23上安装，所以一直保持在这个版本。

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

```plain
POT版本1.31.1
```

### STEPS TO REPRODUCE | 复现步骤

![image](https://github.com/linuxdeepin/developer-center/assets/88034509/c88735da-3cf0-400b-99dd-203fff20c8c8)

### OBSERVED RESULT | 观察到的结果

深度之家用户无法回复

### EXPECTED RESULT | 期望的结果

用户能够回复

```[tasklist]
### Tasks
```

**Comment by BLumia** [comment]
需求请开 discussion 而不是 issue。我把这个 issue 转到 discussion。（edit: 这里是 discussion 了）
